src/views/Search.vue

```diff
@@ -35,7 +35,7 @@ div
       label Use regex
       input(type="checkbox" v-model="use_regex")
     div
-      label Filter AFK
+      label Exclude time away from computer
```

Comment:
"time away from computer" does not reveal what actually happens here. Does it do input-as-active (AFK) and/or audible-as-active? 

I prefer 'Exclude' to 'Filter' though.

Reply:
Resolved in https://github.com/ActivityWatch/aw-webui/pull/291 by adding tooltips